aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -325,12 +325,6 @@ Tensor empty_like(
   // See [Note: hacky wrapper removal for TensorOptions]
   TensorOptions options_ = TensorOptions().dtype(dtype).layout(layout).device(device).pinned_memory(pin_memory);
 
-
-  TORCH_CHECK(
```

Comment:
not sure why this check is being removed ...

Reply:
Options constructed above is guaranteed not to have memory format